astropy/_erfa/setup_package.py

```diff
@@ -64,8 +65,8 @@ def preprocess_source():
         try:
             import jinja2
         except:
-            log.warn("WARNING: jinja2 could not be imported, so the existing "
-                     "ERFA core.py and core.pyx files will be used")
+            warning.warn("WARNING: jinja2 could not be imported, so the existing "
+                     "ERFA core.py and core.pyx files will be used", AstropyWarning)
```

Comment:
This is probably fine, but I need to double check.  Might need to remove `WARNING` from the warning text since it's probably redundant.